### PR TITLE
Add segments update_bulk documentation

### PR DESCRIPTION
**Pull request by pmonfort**

### CODE REVIEW

File: source/includes/cxi/_segments.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+response[custom_properties][][key] | string | name of the segment/property
+response[custom_properties][][value] | string | new value for the segment/property
+
+#### Update bulk
+
+This endpoint updates a list of segments from a response or from a user
+
+```shell
+curl -s -H "Authorization: Bearer ACCESSTOKEN" -X PUT "https://cxi-api.wootric.com/v1/segments/update_bulk" \
+  -F "resposes[id]=RESPONSE_ID" \
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md]: `resposes` or `responses`?

File: source/includes/cxi/_segments.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Segments/Properties
+
+### HTTP Request
+
+`POST https://cxi-api.wootric.com/v1/segments`
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md]: It should be `PUT`, right?

File: source/includes/cxi/_segments.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+users[custom_properties][][value] | string | new value for the segment/property
+
+#### Update bulk
+
+This endpoint updates a list of segments from a response or from a user
+
+```shell
+curl -s -H "Authorization: Bearer ACCESSTOKEN" -X PUT "https://cxi-api.wootric.com/v1/segments/update_bulk" \
+  -F "users[user_id]=USER_ID" \
+  -F "resposes[custom_properties]=[{key: PROPERTY_NAME, value: NEW_VALUE},{key: PROPERTY_NAME, value: NEW_VALUE}]"
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md]: Here should it be `-F "resposes[custom_properties]=[{key: PROPERTY_NAME, value: NEW_VALUE},{key: PROPERTY_NAME, value: NEW_VALUE}]"`
Or `-F "users[custom_properties]=[{key: PROPERTY_NAME, value: NEW_VALUE},{key: PROPERTY_NAME, value: NEW_VALUE}]"`?

File: source/includes/cxi/_segments.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+resposes[id] | integer | Response ID
+response[custom_properties] | Array | list of properties(Hash { key: property_name, value: new_value})
+response[custom_properties][][key] | string | name of the segment/property
+response[custom_properties][][value] | string | new value for the segment/property
+
+#### Update bulk
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md:16]: This is here and in the section below. It seems to be the same text `This endpoint updates a list of segments from a response or from a user`. Is that right?

**Review by diegoserranoa — CHANGES_REQUESTED** [review]
Nice docs! Please take a look at my comments 🙏

File: source/includes/cxi/_segments.md (hunk 1 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md]: The endpoint it's expecting responses 👍

File: source/includes/cxi/_segments.md (hunk 2 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md]: Good catch, thanks

File: source/includes/cxi/_segments.md (hunk 3 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md]: Good catch, thanks

File: source/includes/cxi/_segments.md (hunk 4 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md:16]: Updated, thanks!

**Comment by pmonfort** [comment]
@diegoserranoa thanks for the comments, have made the updates, let me know how it does it looks now

File: source/includes/cxi/_segments.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+### HTTP Request
+
+`PUT https://cxi-api.wootric.com/v1/segments`
+
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+resposes[][id] | integer | Response ID
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md]: `response` 😬

File: source/includes/cxi/_segments.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+response[][custom_properties][][key] | string | name of the segment/property
+response[][custom_properties][][value] | string | new value for the segment/property
+
+#### Update bulk
+
+This endpoint updates a list of segment associated to a response
+
+```shell
+curl -s -H "Authorization: Bearer ACCESSTOKEN" -X PUT "https://cxi-api.wootric.com/v1/segments/update_bulk" \
+  -F "resposes=[id: RESPONSE_ID, custom_properties: [{key: PROPERTY_NAME, value: NEW_VALUE},{key: PROPERTY_NAME, value: NEW_VALUE}]]"
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md]: `responses` :)

**Review by diegoserranoa — CHANGES_REQUESTED** [review]
Found 2 other typos, other than that this is ready to go!

File: source/includes/cxi/_segments.md (hunk 5 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md]: @diegoserranoa Actually, the endpoint is expecting [responses](https://github.com/Wootric/wootric-ai-app-backend/blob/master/app/controllers/v1/segments_controller.rb#L51)

File: source/includes/cxi/_segments.md (hunk 6 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md]: Same

File: source/includes/cxi/_segments.md (hunk 5 above)
Comment (diegoserranoa) [reply source/includes/cxi/_segments.md]: Then it should be `responses`, it's missing the `n`

File: source/includes/cxi/_segments.md (hunk 6 above)
Comment (diegoserranoa) [reply source/includes/cxi/_segments.md]: Same here, it says `-F "resposes=[id: RESPONSE_ID` it should be `-F "responses=[id: RESPONSE_ID`, right?

File: source/includes/cxi/_segments.md (hunk 5 above)
Comment (pmonfort) [reply source/includes/cxi/_segments.md]: facepalm

File: source/includes/cxi/_segments.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+### HTTP Request
+
+`PUT https://cxi-api.wootric.com/v1/segments`
+
+## Response segments bulk update
+
+Parameter | Type | Description
+--------- | ---- | -------
+response[][id] | integer | Response ID
```
Comment (diegoserranoa) [inline source/includes/cxi/_segments.md]: Hahah I'm confused, is it plural or singular? 😅

Comment (pmonfort) [reply source/includes/cxi/_segments.md]: It's plural, thanks for all your comments

**Review by diegoserranoa — APPROVED** [review]
@pmonfort Thanks for addressing my comments! :shipit: :shipit: :shipit:

**Pull request merged**